src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import core.basesyntax.model.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Color[] values = Color.values();
+        return values[new Random().nextInt(values.length)].name();
```

Comment:
make random class-level field
```suggestion
        return values[random.nextInt(values.length)].name();
```